core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -98,6 +99,9 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def useStreamingProcess(): Boolean =
     sqlConf.getConfString(TiConfigConst.COPROCESS_STREAMING, "false").toLowerCase.toBoolean
 
+  private def isEnableChunk(): Boolean =
+    sqlConf.getConfString(TiConfigConst.ENABLE_CHUNK, "true").toLowerCase.toBoolean
```

Comment:
So this is conf on startup only or runtime settable?

Reply:
It means this conf can be set from spark-defaults and runtime. I will remove getter and setter in TiConfiguation.

Reply:
that would be fine.